vars/Amazon.yml

```
@@ -1,4 +1,8 @@
 ---
 # The name of the S3 object corresponding to the CrowdStrike Falcon
 # sensor system package
-package_object_name: falcon-sensor-6.56.0-15309.amzn2023.x86_64.rpm
+package_object_name: falcon-sensor-7.05.0-16004.amzn2023.x86_64.rpm
+
+# The name of the S3 object corresponding to the GPG key for the
+# CrowdStrike Falcon system package
+package_gpg_key_object_name: "Falcon Linux Sensor RPM signing GPG key 2023.gpg"
```

Are the quotes necessary or desired?

```suggestion
package_gpg_key_object_name: Falcon Linux Sensor RPM signing GPG key 2023.gpg
```